### PR TITLE
fix(SEO): add more structured data and meta tag

### PR DESCRIPTION
**Pull request by linooohon**
## Types of Changes

- [x] **Bugfix**
- [ ] **New feature**
- [ ] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [ ] **Other (please describe)**

## Description

- Added meta tag.
- Added missing data.
- Fixed images' alt.

## Related Issue

#102 
#103 

## More Information

- Not sure about <5. Job Posting part>, it is not in this commit so far. I guess needs to be discussed and provide more info.

### CODE REVIEW

File: layouts/default.vue [hunk 1]
```diff
@@ -50,7 +54,13 @@ export default {
                 {
                     hid: 'og:image',
                     property: 'og:image',
-                    content: require('@/static/og-img.jpg'),
+                    content:
+                        'https://tw.pycon.org/2021/_nuxt/img/og-img.bb9ef30.jpg',
```
Comment (mattwang44) [inline layouts/default.vue:58]: the hashed part in the url will be changed after each rebuild.

File: layouts/default.vue [hunk 2]
```diff
@@ -74,11 +84,38 @@ export default {
...
+            script: [
+                {
+                    type: 'application/ld+json',
+                    json: {
+                        '@context': 'https://schema.org',
+                        '@type': 'Organization',
+                        name: 'PyCon Taiwan 2021',
+                        url: 'https://tw.pycon.org/2021/en-us',
+                        logo:
+                            'https://tw.pycon.org/2021/_nuxt/img/og-img.bb9ef30.jpg',
```
Comment (mattwang44) [inline layouts/default.vue:116]: the same

**Comment by mattwang44** [comment]
- We can implement job listing part in another PR since we have not received any data yet.
- Please rebase to the latest main branch and add `head()` for those pages without it.

File: layouts/default.vue [hunk 3]
```diff
@@ -74,11 +84,38 @@ export default {
...
+                },
+            ],
+            script: [
+                {
+                    type: 'application/ld+json',
+                    json: {
+                        '@context': 'https://schema.org',
+                        '@type': 'Organization',
+                        name: 'PyCon Taiwan 2021',
+                        url: 'https://tw.pycon.org/2021/en-us',
```
Comment (mattwang44) [inline layouts/default.vue:114]:
```suggestion
                        name: 'PyCon Taiwan',
                        url: 'https://tw.pycon.org',
```

File: layouts/default.vue [hunk 4]
```diff
@@ -74,11 +84,38 @@ export default {
...
+                    hreflang: 'en-us',
+                    href: 'https://tw.pycon.org/2021/en-us',
+                },
+                {
+                    rel: 'alternate',
+                    hreflang: 'zh-hant-tw',
+                    href: 'https://tw.pycon.org/2021/zh-hant',
+                },
+            ],
+            script: [
```
Comment (mattwang44) [inline layouts/default.vue:107]: just for the record, these sites could be useful
- https://search.google.com/test/rich-results
- https://schema.org/Organization

File: layouts/default.vue [hunk 5]
```diff
@@ -50,7 +54,13 @@ export default {
...
                     hid: 'og:image',
                     property: 'og:image',
-                    content: require('@/static/og-img.jpg'),
+                    content:
+                        'https://tw.pycon.org/2021/_nuxt/img/og-img.bb9ef30.jpg',
+                },
+                {
+                    hid: 'og:url',
+                    property: 'og:url',
+                    content: 'https://tw.pycon.org' + this.$route.path,
```
Comment (mattwang44) [inline layouts/default.vue:63]:
```suggestion
                    content: 'https://tw.pycon.org/2021' + this.$route.path,
```

File: layouts/default.vue (hunk 3 above)
Comment (mattwang44) [inline layouts/default.vue:114]: Please also add [`sameAs`](https://schema.org/sameAs).
```suggestion
                        url: 'https://tw.pycon.org/2021/en-us',
                        sameAs: [ 
                            "https://www.facebook.com/pycontw/",
                            "https://twitter.com/PyConTW/",
                            "https://www.linkedin.com/company/pycontw",
                            "https://www.youtube.com/PyConTaiwan",
                            "https://instagram.com/pycontw/",
                            "https://github.com/pycontw/",
                            "https://pycontw.blogspot.com/",
                        ],
}
```

**Comment by linooohon** [comment]
@mattwang44 ,Fixed the structure and also rebase to the latest main branch.

**Review by mattwang44 — APPROVED** [review]
LGTM

**Pull request merged**